python/dictionaries.md

```diff
@@ -68,14 +77,17 @@ We can create dictionaries from lists of 2-item lists.
 {0: 'a thing', 1: 'another'}
 ```
 
-And also with _dictionary comprehensions_, in a similar manner to list 
+And also with _dictionary comprehensions_, in a similar manner to list
 comprehensions, with the additional specification of the key.
 
 ```python
 >>> {i: i**i for i in range(5) if i != 3}
 {0: 1, 1: 1, 2: 4, 4: 256, 5: 3125}
 ```
 
+Note that dictionary comprehensions require at least Python 2.7 to work.
```

Comment:
Do we think compatibility with 2.6 is important?

Reply:
It depends. I am just thinking about the poor man connecting to lxplus, trying dictionary comprehensions and seeing that they don't work for some reason because she's forgotten to load the correct environment. I don't think this note makes much harm :-)

Reply:
Yeah, it's really annoying the default for `ssh lxplus` users is still 2.6... OK, let's keep it.